mapie/quantile_regression.py

```diff
@@ -316,13 +321,14 @@ def _check_cv(
                 "Invalid cv method, only valid method is ``split``."
             )
 
-    def fit(  # type: ignore
+    def fit(
         self,
         X: ArrayLike,
         y: ArrayLike,
-        X_calib: ArrayLike,
-        y_calib: ArrayLike,
         sample_weight: Optional[ArrayLike] = None,
+        X_calib: Optional[ArrayLike] = None,
```

Comment:
Rearrange the list of arguments in the docstring accordingly. 
Add `calib_size` in docstring.

Reply:
yes, done